THIS IS MY #3 Pull Req

byam opened the pull request.

byam committed:
THIS IS MY #3 Pull Req

byam committed:
SECOND COMMIT

byam committed:
THIRD COMMIT

bot-mask commented:
Hello Guys. I am Mr.Robot. 

```
GitHub pull request #4 of commit 6b7a33d3b34b7913e0dc7b103c95cfdec5189fde, no merge conflicts.
Setting status of 6b7a33d3b34b7913e0dc7b103c95cfdec5189fde to PENDING with url http://10.15.247.135:8081/job/ghpqb-sql-test/34/ and message: "Build started sha1 is merged."
Setting status of 6b7a33d3b34b7913e0dc7b103c95cfdec5189fde to PENDING with url http://10.15.247.135:8081/job/ghpqb-sql-test/34/ and message: "Build started sha1 is merged."
ビルドします。 ワークスペース: /usr/local/opt/jenkins/jobs/ghpqb-sql-test/workspace
 > git rev-parse --is-inside-work-tree # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url https://github.com/boyd-mask/git-jenkins-test/ # timeout=10
Fetching upstream changes from https://github.com/boyd-mask/git-jenkins-test/
 > git --version # timeout=10
 > git fetch --tags --progress https://github.com/boyd-mask/git-jenkins-test/ +refs/heads/*:refs/remotes/origin/* +refs/pull/*:refs/remotes/origin/pr/*
 > git rev-parse refs/remotes/origin/pr/4/merge^{commit} # timeout=10
 > git rev-parse refs/remotes/origin/origin/pr/4/merge^{commit} # timeout=10
Checking out Revision 15e9285549a339514d28b5188304e76d63fc8d4d (refs/remotes/origin/pr/4/merge)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f 15e9285549a339514d28b5188304e76d63fc8d4d
First time build. Skipping changelog.
[workspace] $ /bin/sh -xe /tmp/hudson5093126375816775146.sh
+ /usr/local/bin/fab -f /usr/local/jenkins/fabfile.py CurrentCommitMessage:oauth_token=6d85708aa08c6d12c6fedab3b600424ada445df3,github_api_url=https://api.github.com,owner=boyd-mask,repo=git-jenkins-test,git_current_branch=origin/pr/4/merge,git_current_commit_id=15e9285549a339514d28b5188304e76d63fc8d4d
[localhost] local: curl -H "Authorization: token 6d85708aa08c6d12c6fedab3b600424ada445df3" https://api.github.com/repos/boyd-mask/git-jenkins-test/commits/15e9285549a339514d28b5188304e76d63fc8d4d
Traceback (most recent call last):
  File "/usr/local/Python-2.7.9/lib/python2.7/site-packages/fabric/main.py", line 743, in main
    *args, **kwargs
  File "/usr/local/Python-2.7.9/lib/python2.7/site-packages/fabric/tasks.py", line 405, in execute
    results["<local-only>"] = task.run(*args, **new_kwargs)
  File "/usr/local/Python-2.7.9/lib/python2.7/site-packages/fabric/tasks.py", line 171, in run
    return self.wrapped(*args, **kwargs)
  File "/usr/local/jenkins/fabfile.py", line 19, in CurrentCommitMessage
    git_head_commit_id = json.loads(str(commit_raw))["parents"][-1]["sha"]
KeyError: "parents"
Build step "シェルの実行" marked build as failure
Adding one-line test results to commit status...
Setting status of 6b7a33d3b34b7913e0dc7b103c95cfdec5189fde to FAILURE with url http://10.15.247.135:8081/job/ghpqb-sql-test/34/ and message: "FAILED!
 No test results found."
Setting status of 6b7a33d3b34b7913e0dc7b103c95cfdec5189fde to FAILURE with url http://10.15.247.135:8081/job/ghpqb-sql-test/34/ and message: "failure
 "
Warning: you have no plugins providing access control for builds, so falling back to legacy behavior of permitting any downstream builds to be triggered
ha:AAAApx+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAzREgZp/az8JP3k/Nzc1LwS3aLU4tKcEt30zJKM0iR9APyWVhPQAAAAcomment-result-github の新規ビルドを実行します
Finished: FAILURE

```

byam committed:
FOURTH COMMIT

byam committed:
FIFTH COMMIT

bot-mask commented:
Hello Guys. I am Mr.Robot. 

```
GitHub pull request #4 of commit 3b7d59ed18632915f4280850228e153a449f7c9e, no merge conflicts.
Setting status of 3b7d59ed18632915f4280850228e153a449f7c9e to PENDING with url http://10.15.247.135:8081/job/ghpqb-sql-test/36/ and message: "Build started sha1 is merged."
Setting status of 3b7d59ed18632915f4280850228e153a449f7c9e to PENDING with url http://10.15.247.135:8081/job/ghpqb-sql-test/36/ and message: "Build started sha1 is merged."
ビルドします。 ワークスペース: /usr/local/opt/jenkins/jobs/ghpqb-sql-test/workspace
 > git rev-parse --is-inside-work-tree # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url https://github.com/boyd-mask/git-jenkins-test/ # timeout=10
Fetching upstream changes from https://github.com/boyd-mask/git-jenkins-test/
 > git --version # timeout=10
 > git fetch --tags --progress https://github.com/boyd-mask/git-jenkins-test/ +refs/heads/*:refs/remotes/origin/* +refs/pull/*:refs/remotes/origin/pr/*
 > git rev-parse refs/remotes/origin/pr/4/merge^{commit} # timeout=10
 > git rev-parse refs/remotes/origin/origin/pr/4/merge^{commit} # timeout=10
Checking out Revision 2d4e922dd696cd8647e167641753271a20ce8ee6 (refs/remotes/origin/pr/4/merge)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f 2d4e922dd696cd8647e167641753271a20ce8ee6
First time build. Skipping changelog.
[workspace] $ /bin/sh -xe /tmp/hudson3002490471348932838.sh
+ /usr/local/bin/fab -f /usr/local/jenkins/fabfile.py CurrentCommitMessage:oauth_token=42b1fb7c1b06f7e095492adc82d64c57a5fdef73,github_api_url=https://api.github.com,owner=boyd-mask,repo=git-jenkins-test,git_current_branch=origin/pr/4/merge,git_current_commit_id=2d4e922dd696cd8647e167641753271a20ce8ee6
[localhost] local: curl -H "Authorization: token 42b1fb7c1b06f7e095492adc82d64c57a5fdef73" https://api.github.com/repos/boyd-mask/git-jenkins-test/commits/2d4e922dd696cd8647e167641753271a20ce8ee6
git_head_commit_id: 3b7d59ed18632915f4280850228e153a449f7c9e
[localhost] local: curl -H "Authorization: token 42b1fb7c1b06f7e095492adc82d64c57a5fdef73" https://api.github.com/repos/boyd-mask/git-jenkins-test/commits/3b7d59ed18632915f4280850228e153a449f7c9e

Done.
Adding one-line test results to commit status...
Setting status of 3b7d59ed18632915f4280850228e153a449f7c9e to SUCCESS with url http://10.15.247.135:8081/job/ghpqb-sql-test/36/ and message: "YEEEEEEEEESSSSSSSS!
 No test results found."
Setting status of 3b7d59ed18632915f4280850228e153a449f7c9e to SUCCESS with url http://10.15.247.135:8081/job/ghpqb-sql-test/36/ and message: "success
 "
Warning: you have no plugins providing access control for builds, so falling back to legacy behavior of permitting any downstream builds to be triggered
[8mha:AAAApx+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAzREgZp/az8JP3k/Nzc1LwS3aLU4tKcEt30zJKM0iR9APyWVhPQAAAA[0mcomment-result-github の新規ビルドを実行します
Finished: SUCCESS
```

byam committed:
SIXTH COMMIT

bot-mask commented:
Hello Guys. I am Mr.Robot. 

```
GitHub pull request #4 of commit 358c3491089105647a181b250c07bc197eb4f6a3, no merge conflicts.
Setting status of 358c3491089105647a181b250c07bc197eb4f6a3 to PENDING with url http://10.15.247.135:8081/job/ghpqb-sql-test/37/ and message: "Build started sha1 is merged."
Setting status of 358c3491089105647a181b250c07bc197eb4f6a3 to PENDING with url http://10.15.247.135:8081/job/ghpqb-sql-test/37/ and message: "Build started sha1 is merged."
ビルドします。 ワークスペース: /usr/local/opt/jenkins/jobs/ghpqb-sql-test/workspace
 > git rev-parse --is-inside-work-tree # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url https://github.com/boyd-mask/git-jenkins-test/ # timeout=10
Fetching upstream changes from https://github.com/boyd-mask/git-jenkins-test/
 > git --version # timeout=10
 > git fetch --tags --progress https://github.com/boyd-mask/git-jenkins-test/ +refs/heads/*:refs/remotes/origin/* +refs/pull/*:refs/remotes/origin/pr/*
 > git rev-parse refs/remotes/origin/pr/4/merge^{commit} # timeout=10
 > git rev-parse refs/remotes/origin/origin/pr/4/merge^{commit} # timeout=10
Checking out Revision 9f57d014951c859cdaeeaef8f0d45ed00713678d (refs/remotes/origin/pr/4/merge)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f 9f57d014951c859cdaeeaef8f0d45ed00713678d
First time build. Skipping changelog.
[workspace] $ /bin/sh -xe /tmp/hudson5148382349436345913.sh
+ /usr/local/bin/fab -f /usr/local/jenkins/fabfile.py CurrentCommitMessage:oauth_token=42b1fb7c1b06f7e095492adc82d64c57a5fdef73,github_api_url=https://api.github.com,owner=boyd-mask,repo=git-jenkins-test,git_current_branch=origin/pr/4/merge,git_current_commit_id=9f57d014951c859cdaeeaef8f0d45ed00713678d
[localhost] local: curl -H "Authorization: token 42b1fb7c1b06f7e095492adc82d64c57a5fdef73" https://api.github.com/repos/boyd-mask/git-jenkins-test/commits/9f57d014951c859cdaeeaef8f0d45ed00713678d
git_head_commit_id: 358c3491089105647a181b250c07bc197eb4f6a3
[localhost] local: curl -H "Authorization: token 42b1fb7c1b06f7e095492adc82d64c57a5fdef73" https://api.github.com/repos/boyd-mask/git-jenkins-test/commits/358c3491089105647a181b250c07bc197eb4f6a3
SIXTH COMMIT

Done.
Adding one-line test results to commit status...
Setting status of 358c3491089105647a181b250c07bc197eb4f6a3 to SUCCESS with url http://10.15.247.135:8081/job/ghpqb-sql-test/37/ and message: "YEEEEEEEEESSSSSSSS!
 No test results found."
Setting status of 358c3491089105647a181b250c07bc197eb4f6a3 to SUCCESS with url http://10.15.247.135:8081/job/ghpqb-sql-test/37/ and message: "success
 "
Warning: you have no plugins providing access control for builds, so falling back to legacy behavior of permitting any downstream builds to be triggered
[8mha:AAAApx+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAzREgZp/az8JP3k/Nzc1LwS3aLU4tKcEt30zJKM0iR9APyWVhPQAAAA[0mcomment-result-github の新規ビルドを実行します
Finished: SUCCESS
```

byam committed:
SEVENTH COMMIT

byam committed:
EIGHT COMMIT

bot-mask commented:
Hello Guys. I am Mr.Robot. 

```
GitHub pull request #4 of commit d33be1c3f7db70ae7563a07fccc5ad38e661e1cc, no merge conflicts.
Setting status of d33be1c3f7db70ae7563a07fccc5ad38e661e1cc to PENDING with url http://10.15.247.135:8081/job/ghpqb-sql-test/39/ and message: "Build started sha1 is merged."
Setting status of d33be1c3f7db70ae7563a07fccc5ad38e661e1cc to PENDING with url http://10.15.247.135:8081/job/ghpqb-sql-test/39/ and message: "Build started sha1 is merged."
ビルドします。 ワークスペース: /usr/local/opt/jenkins/jobs/ghpqb-sql-test/workspace
 > git rev-parse --is-inside-work-tree # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url https://github.com/boyd-mask/git-jenkins-test/ # timeout=10
Fetching upstream changes from https://github.com/boyd-mask/git-jenkins-test/
 > git --version # timeout=10
 > git fetch --tags --progress https://github.com/boyd-mask/git-jenkins-test/ +refs/heads/*:refs/remotes/origin/* +refs/pull/*:refs/remotes/origin/pr/*
 > git rev-parse refs/remotes/origin/pr/4/merge^{commit} # timeout=10
 > git rev-parse refs/remotes/origin/origin/pr/4/merge^{commit} # timeout=10
Checking out Revision f6d81f36ca03ebe6a7c0ab3050d3dfacd9db89b3 (refs/remotes/origin/pr/4/merge)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f f6d81f36ca03ebe6a7c0ab3050d3dfacd9db89b3
First time build. Skipping changelog.
[workspace] $ /bin/sh -xe /tmp/hudson4438620719774939634.sh
+ /usr/local/bin/fab -f /usr/local/jenkins/fabfile.py CurrentCommitMessage:oauth_token=********,github_api_url=https://api.github.com,owner=boyd-mask,repo=git-jenkins-test,git_current_branch=origin/pr/4/merge,git_current_commit_id=f6d81f36ca03ebe6a7c0ab3050d3dfacd9db89b3
[localhost] local: curl -H "Authorization: token ********" https://api.github.com/repos/boyd-mask/git-jenkins-test/commits/f6d81f36ca03ebe6a7c0ab3050d3dfacd9db89b3
git_head_commit_id: d33be1c3f7db70ae7563a07fccc5ad38e661e1cc
[localhost] local: curl -H "Authorization: token ********" https://api.github.com/repos/boyd-mask/git-jenkins-test/commits/d33be1c3f7db70ae7563a07fccc5ad38e661e1cc
EIGHT COMMIT

Done.
Adding one-line test results to commit status...
Setting status of d33be1c3f7db70ae7563a07fccc5ad38e661e1cc to SUCCESS with url http://10.15.247.135:8081/job/ghpqb-sql-test/39/ and message: "YEEEEEEEEESSSSSSSS!
 No test results found."
Setting status of d33be1c3f7db70ae7563a07fccc5ad38e661e1cc to SUCCESS with url http://10.15.247.135:8081/job/ghpqb-sql-test/39/ and message: "success
 "
Warning: you have no plugins providing access control for builds, so falling back to legacy behavior of permitting any downstream builds to be triggered
[8mha:AAAApx+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAzREgZp/az8JP3k/Nzc1LwS3aLU4tKcEt30zJKM0iR9APyWVhPQAAAA[0mcomment-result-github の新規ビルドを実行します
Finished: SUCCESS
```

byam committed:
/usr/local/opt/jenkins/jobs/ghpqb-sql-test/workspace/.git/FETCH_HEAD
/usr/local/opt/jenkins/jobs/ghpqb-sql-test/workspace/

bot-mask commented:
Hello Guys. I am Mr.Robot. 

```
GitHub pull request #4 of commit 1058ea5a6d9e34337cce2829f943774c4bfc5f51, no merge conflicts.
Setting status of 1058ea5a6d9e34337cce2829f943774c4bfc5f51 to PENDING with url http://10.15.247.135:8081/job/ghpqb-sql-test/40/ and message: "Build started sha1 is merged."
Setting status of 1058ea5a6d9e34337cce2829f943774c4bfc5f51 to PENDING with url http://10.15.247.135:8081/job/ghpqb-sql-test/40/ and message: "Build started sha1 is merged."
ビルドします。 ワークスペース: /usr/local/opt/jenkins/jobs/ghpqb-sql-test/workspace
 > git rev-parse --is-inside-work-tree # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url https://github.com/boyd-mask/git-jenkins-test/ # timeout=10
Fetching upstream changes from https://github.com/boyd-mask/git-jenkins-test/
 > git --version # timeout=10
 > git fetch --tags --progress https://github.com/boyd-mask/git-jenkins-test/ +refs/heads/*:refs/remotes/origin/* +refs/pull/*:refs/remotes/origin/pr/*
 > git rev-parse refs/remotes/origin/pr/4/merge^{commit} # timeout=10
 > git rev-parse refs/remotes/origin/origin/pr/4/merge^{commit} # timeout=10
Checking out Revision e77dd1e68c2716e55892e0e5a95e2e4306d83ce3 (refs/remotes/origin/pr/4/merge)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f e77dd1e68c2716e55892e0e5a95e2e4306d83ce3
First time build. Skipping changelog.
[workspace] $ /bin/sh -xe /tmp/hudson5006562845854429940.sh
+ /usr/local/bin/fab -f /usr/local/jenkins/fabfile.py CurrentCommitMessage:oauth_token=********,github_api_url=https://api.github.com,owner=boyd-mask,repo=git-jenkins-test,git_current_branch=origin/pr/4/merge,git_current_commit_id=e77dd1e68c2716e55892e0e5a95e2e4306d83ce3
[localhost] local: curl -H "Authorization: token ********" https://api.github.com/repos/boyd-mask/git-jenkins-test/commits/e77dd1e68c2716e55892e0e5a95e2e4306d83ce3
git_head_commit_id: 1058ea5a6d9e34337cce2829f943774c4bfc5f51
[localhost] local: curl -H "Authorization: token ********" https://api.github.com/repos/boyd-mask/git-jenkins-test/commits/1058ea5a6d9e34337cce2829f943774c4bfc5f51
/usr/local/opt/jenkins/jobs/ghpqb-sql-test/workspace/.git/FETCH_HEAD
/usr/local/opt/jenkins/jobs/ghpqb-sql-test/workspace/

Done.
Adding one-line test results to commit status...
Setting status of 1058ea5a6d9e34337cce2829f943774c4bfc5f51 to SUCCESS with url http://10.15.247.135:8081/job/ghpqb-sql-test/40/ and message: "YEEEEEEEEESSSSSSSS!
 No test results found."
Setting status of 1058ea5a6d9e34337cce2829f943774c4bfc5f51 to SUCCESS with url http://10.15.247.135:8081/job/ghpqb-sql-test/40/ and message: "success
 "
Warning: you have no plugins providing access control for builds, so falling back to legacy behavior of permitting any downstream builds to be triggered
[8mha:AAAApx+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAzREgZp/az8JP3k/Nzc1LwS3aLU4tKcEt30zJKM0iR9APyWVhPQAAAA[0mcomment-result-github の新規ビルドを実行します
Finished: SUCCESS
```

byam committed:
YAML'S ORDERS
/usr/local/opt/jenkins/jobs/ghpqb-sql-test/workspace/
/usr/local/opt/jenkins/jobs/comment-result-github/workspace/

bot-mask commented:
Hello Guys. I am Mr.Robot. 

```
GitHub pull request #4 of commit bd088eff9ee794802634297ee16ae5647b5b0f89, no merge conflicts.
Setting status of bd088eff9ee794802634297ee16ae5647b5b0f89 to PENDING with url http://10.15.247.135:8081/job/ghpqb-sql-test/41/ and message: "Build started sha1 is merged."
Setting status of bd088eff9ee794802634297ee16ae5647b5b0f89 to PENDING with url http://10.15.247.135:8081/job/ghpqb-sql-test/41/ and message: "Build started sha1 is merged."
ビルドします。 ワークスペース: /usr/local/opt/jenkins/jobs/ghpqb-sql-test/workspace
 > git rev-parse --is-inside-work-tree # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url https://github.com/boyd-mask/git-jenkins-test/ # timeout=10
Fetching upstream changes from https://github.com/boyd-mask/git-jenkins-test/
 > git --version # timeout=10
 > git fetch --tags --progress https://github.com/boyd-mask/git-jenkins-test/ +refs/heads/*:refs/remotes/origin/* +refs/pull/*:refs/remotes/origin/pr/*
 > git rev-parse refs/remotes/origin/pr/4/merge^{commit} # timeout=10
 > git rev-parse refs/remotes/origin/origin/pr/4/merge^{commit} # timeout=10
Checking out Revision da7066918f6c02e8166b44992976a075e972f3a7 (refs/remotes/origin/pr/4/merge)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f da7066918f6c02e8166b44992976a075e972f3a7
First time build. Skipping changelog.
[workspace] $ /bin/sh -xe /tmp/hudson7756008322138019861.sh
+ /usr/local/bin/fab -f /usr/local/jenkins/fabfile.py RunYaml:oauth_token=********,github_api_url=https://api.github.com,owner=boyd-mask,repo=git-jenkins-test,git_current_branch=origin/pr/4/merge,git_current_commit_id=da7066918f6c02e8166b44992976a075e972f3a7
README.md
debug.ipynb
env_vars
git_envs
hi
index.html
log
[localhost] local: curl -H "Authorization: token ********" https://api.github.com/repos/boyd-mask/git-jenkins-test/commits/da7066918f6c02e8166b44992976a075e972f3a7
git_head_commit_id: bd088eff9ee794802634297ee16ae5647b5b0f89
[localhost] local: curl -H "Authorization: token ********" https://api.github.com/repos/boyd-mask/git-jenkins-test/commits/bd088eff9ee794802634297ee16ae5647b5b0f89
YAML"S ORDERS
/usr/local/opt/jenkins/jobs/ghpqb-sql-test/workspace/
/usr/local/opt/jenkins/jobs/comment-result-github/workspace/
[localhost] local: ls /usr/local/opt/jenkins/jobs/ghpqb-sql-test/workspace/

[localhost] local: ls /usr/local/opt/jenkins/jobs/comment-result-github/workspace/


Done.
Adding one-line test results to commit status...
Setting status of bd088eff9ee794802634297ee16ae5647b5b0f89 to SUCCESS with url http://10.15.247.135:8081/job/ghpqb-sql-test/41/ and message: "YEEEEEEEEESSSSSSSS!
 No test results found."
Setting status of bd088eff9ee794802634297ee16ae5647b5b0f89 to SUCCESS with url http://10.15.247.135:8081/job/ghpqb-sql-test/41/ and message: "success
 "
Warning: you have no plugins providing access control for builds, so falling back to legacy behavior of permitting any downstream builds to be triggered
[8mha:AAAApx+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAzREgZp/az8JP3k/Nzc1LwS3aLU4tKcEt30zJKM0iR9APyWVhPQAAAA[0mcomment-result-github の新規ビルドを実行します
Finished: SUCCESS
```

byam committed:
YAML TEST BASIC ORDERS
/usr/local/opt/jenkins/jobs/comment-result-github
/usr/local/opt/jenkins/jobs/ghpqb-sql-test/workspace/
/usr/local/opt/jenkins/

bot-mask commented:
Hello Guys. I am Mr.Robot. 

```
GitHub pull request #4 of commit fe6a82a385ca62c2efdf7e22e9d640502060546f, no merge conflicts.
Setting status of fe6a82a385ca62c2efdf7e22e9d640502060546f to PENDING with url http://10.15.247.135:8081/job/ghpqb-sql-test/42/ and message: "Build started sha1 is merged."
Setting status of fe6a82a385ca62c2efdf7e22e9d640502060546f to PENDING with url http://10.15.247.135:8081/job/ghpqb-sql-test/42/ and message: "Build started sha1 is merged."
ビルドします。 ワークスペース: /usr/local/opt/jenkins/jobs/ghpqb-sql-test/workspace
 > git rev-parse --is-inside-work-tree # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url https://github.com/boyd-mask/git-jenkins-test/ # timeout=10
Fetching upstream changes from https://github.com/boyd-mask/git-jenkins-test/
 > git --version # timeout=10
 > git fetch --tags --progress https://github.com/boyd-mask/git-jenkins-test/ +refs/heads/*:refs/remotes/origin/* +refs/pull/*:refs/remotes/origin/pr/*
 > git rev-parse refs/remotes/origin/pr/4/merge^{commit} # timeout=10
 > git rev-parse refs/remotes/origin/origin/pr/4/merge^{commit} # timeout=10
Checking out Revision e7acf8dcb7dcdbee5eceb79ac67238041fe49fbf (refs/remotes/origin/pr/4/merge)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f e7acf8dcb7dcdbee5eceb79ac67238041fe49fbf
First time build. Skipping changelog.
[workspace] $ /bin/sh -xe /tmp/hudson3312184642105238548.sh
+ /usr/local/bin/fab -f /usr/local/jenkins/fabfile.py RunYaml:oauth_token=********,github_api_url=https://api.github.com,owner=boyd-mask,repo=git-jenkins-test,git_current_branch=origin/pr/4/merge,git_current_commit_id=e7acf8dcb7dcdbee5eceb79ac67238041fe49fbf
builds
config.xml
lastStable
lastSuccessful
nextBuildNumber
workspace
README.md
debug.ipynb
env_vars
git_envs
hi
index.html
log
bin
com.michelin.cio.hudson.plugins.maskpasswords.MaskPasswordsConfig.xml
config.xml
credentials.xml
github-plugin-configuration.xml
hudson.maven.MavenModuleSet.xml
hudson.model.UpdateCenter.xml
hudson.plugins.build_timeout.operations.BuildStepOperation.xml
hudson.plugins.emailext.ExtendedEmailPublisher.xml
hudson.plugins.git.GitSCM.xml
hudson.plugins.git.GitTool.xml
hudson.plugins.timestamper.TimestamperConfig.xml
hudson.scm.SubversionSCM.xml
hudson.tasks.Mailer.xml
hudson.tasks.Shell.xml
hudson.triggers.SCMTrigger.xml
identity.key.enc
jenkins-cli.jar
jenkins.install.InstallUtil.lastExecVersion
jenkins.install.UpgradeWizard.state
jenkins.model.ArtifactManagerConfiguration.xml
jenkins.model.DownloadSettings.xml
jenkins.model.JenkinsLocationConfiguration.xml
jenkins.security.QueueItemAuthenticatorConfiguration.xml
jobs
logs
nodeMonitors.xml
nodes
org.jenkinsci.plugins.conditionalbuildstep.singlestep.SingleConditionalBuilder.xml
org.jenkinsci.plugins.ghprb.GhprbTrigger.xml
plugins
queue.xml.bak
secret.key
secret.key.not-so-secret
secrets
updates
userContent
users
war
workflow-libs
[localhost] local: curl -H "Authorization: token ********" https://api.github.com/repos/boyd-mask/git-jenkins-test/commits/e7acf8dcb7dcdbee5eceb79ac67238041fe49fbf
git_head_commit_id: fe6a82a385ca62c2efdf7e22e9d640502060546f
[localhost] local: curl -H "Authorization: token ********" https://api.github.com/repos/boyd-mask/git-jenkins-test/commits/fe6a82a385ca62c2efdf7e22e9d640502060546f
YAML TEST BASIC ORDERS
/usr/local/opt/jenkins/jobs/comment-result-github
/usr/local/opt/jenkins/jobs/ghpqb-sql-test/workspace/
/usr/local/opt/jenkins/
[localhost] local: ls /usr/local/opt/jenkins/jobs/comment-result-github
/usr/local/opt/jenkins/jobs/comment-result-github

[localhost] local: ls /usr/local/opt/jenkins/jobs/ghpqb-sql-test/workspace/
/usr/local/opt/jenkins/jobs/ghpqb-sql-test/workspace/

[localhost] local: ls /usr/local/opt/jenkins/
/usr/local/opt/jenkins/


Done.
Adding one-line test results to commit status...
Setting status of fe6a82a385ca62c2efdf7e22e9d640502060546f to SUCCESS with url http://10.15.247.135:8081/job/ghpqb-sql-test/42/ and message: "YEEEEEEEEESSSSSSSS!
 No test results found."
Setting status of fe6a82a385ca62c2efdf7e22e9d640502060546f to SUCCESS with url http://10.15.247.135:8081/job/ghpqb-sql-test/42/ and message: "success
 "
Warning: you have no plugins providing access control for builds, so falling back to legacy behavior of permitting any downstream builds to be triggered
[8mha:AAAApx+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAzREgZp/az8JP3k/Nzc1LwS3aLU4tKcEt30zJKM0iR9APyWVhPQAAAA[0mcomment-result-github の新規ビルドを実行します
Finished: SUCCESS
```

byam committed:
HEY YO!!!!!!!!!!!
/usr/local/
/usr/local/opt
/usr/local/opt/jenkins

byam commented:
bot test this

bot-mask commented:
Hello Guys. I am Mr.Robot. 

```
GitHub pull request #4 of commit 85f427d6b0bf307a0b3a1e3eb95dda3d799a1bcc, no merge conflicts.
Setting status of 85f427d6b0bf307a0b3a1e3eb95dda3d799a1bcc to PENDING with url http://10.15.247.135:8081/job/ghpqb-sql-test/43/ and message: "Build started sha1 is merged."
Setting status of 85f427d6b0bf307a0b3a1e3eb95dda3d799a1bcc to PENDING with url http://10.15.247.135:8081/job/ghpqb-sql-test/43/ and message: "Build started sha1 is merged."
ビルドします。 ワークスペース: /usr/local/opt/jenkins/jobs/ghpqb-sql-test/workspace
 > git rev-parse --is-inside-work-tree # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url https://github.com/boyd-mask/git-jenkins-test/ # timeout=10
Fetching upstream changes from https://github.com/boyd-mask/git-jenkins-test/
 > git --version # timeout=10
 > git fetch --tags --progress https://github.com/boyd-mask/git-jenkins-test/ +refs/heads/*:refs/remotes/origin/* +refs/pull/*:refs/remotes/origin/pr/*
 > git rev-parse refs/remotes/origin/pr/4/merge^{commit} # timeout=10
 > git rev-parse refs/remotes/origin/origin/pr/4/merge^{commit} # timeout=10
Checking out Revision ada657dfcb789b02e653714123c065d462bdcf8f (refs/remotes/origin/pr/4/merge)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f ada657dfcb789b02e653714123c065d462bdcf8f
First time build. Skipping changelog.
[workspace] $ /bin/sh -xe /tmp/hudson537813289726905937.sh
+ /usr/local/bin/fab -f /usr/local/jenkins/fabfile.py RunYaml:oauth_token=********,github_api_url=https://api.github.com,owner=boyd-mask,repo=git-jenkins-test,git_current_branch=origin/pr/4/merge,git_current_commit_id=ada657dfcb789b02e653714123c065d462bdcf8f
Python-2.7.9
bin
etc
games
include
jenkins
lib
lib64
libexec
nz
opt
ruby-2.2.0
sbin
share
src
jenkins
bin
com.michelin.cio.hudson.plugins.maskpasswords.MaskPasswordsConfig.xml
config.xml
credentials.xml
github-plugin-configuration.xml
hudson.maven.MavenModuleSet.xml
hudson.model.UpdateCenter.xml
hudson.plugins.build_timeout.operations.BuildStepOperation.xml
hudson.plugins.emailext.ExtendedEmailPublisher.xml
hudson.plugins.git.GitSCM.xml
hudson.plugins.git.GitTool.xml
hudson.plugins.timestamper.TimestamperConfig.xml
hudson.scm.SubversionSCM.xml
hudson.tasks.Mailer.xml
hudson.tasks.Shell.xml
hudson.triggers.SCMTrigger.xml
identity.key.enc
jenkins-cli.jar
jenkins.install.InstallUtil.lastExecVersion
jenkins.install.UpgradeWizard.state
jenkins.model.ArtifactManagerConfiguration.xml
jenkins.model.DownloadSettings.xml
jenkins.model.JenkinsLocationConfiguration.xml
jenkins.security.QueueItemAuthenticatorConfiguration.xml
jobs
logs
nodeMonitors.xml
nodes
org.jenkinsci.plugins.conditionalbuildstep.singlestep.SingleConditionalBuilder.xml
org.jenkinsci.plugins.ghprb.GhprbTrigger.xml
plugins
queue.xml.bak
secret.key
secret.key.not-so-secret
secrets
updates
userContent
users
war
workflow-libs
[localhost] local: curl -H "Authorization: token ********" https://api.github.com/repos/boyd-mask/git-jenkins-test/commits/ada657dfcb789b02e653714123c065d462bdcf8f
git_head_commit_id: 85f427d6b0bf307a0b3a1e3eb95dda3d799a1bcc
[localhost] local: curl -H "Authorization: token ********" https://api.github.com/repos/boyd-mask/git-jenkins-test/commits/85f427d6b0bf307a0b3a1e3eb95dda3d799a1bcc
HEY YO!!!!!!!!!!!
/usr/local/
/usr/local/opt
/usr/local/opt/jenkins
[localhost] local: ls /usr/local/
/usr/local/

[localhost] local: ls /usr/local/opt
/usr/local/opt

[localhost] local: ls /usr/local/opt/jenkins
/usr/local/opt/jenkins


Done.
Adding one-line test results to commit status...
Setting status of 85f427d6b0bf307a0b3a1e3eb95dda3d799a1bcc to SUCCESS with url http://10.15.247.135:8081/job/ghpqb-sql-test/43/ and message: "YEEEEEEEEESSSSSSSS!
 No test results found."
Setting status of 85f427d6b0bf307a0b3a1e3eb95dda3d799a1bcc to SUCCESS with url http://10.15.247.135:8081/job/ghpqb-sql-test/43/ and message: " "
Warning: you have no plugins providing access control for builds, so falling back to legacy behavior of permitting any downstream builds to be triggered
[8mha:AAAApx+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAzREgZp/az8JP3k/Nzc1LwS3aLU4tKcEt30zJKM0iR9APyWVhPQAAAA[0mcomment-result-github の新規ビルドを実行します
Finished: SUCCESS
```

byam committed:
TEST COMMIT
/usr/local

byam commented:
bot test this

bot-mask commented:
Hello Guys. I am Mr.Robot. 

```
GitHub pull request #4 of commit 716511a912e57e383299fb1777d9dfc9ea1e30ff, no merge conflicts.
Setting status of 716511a912e57e383299fb1777d9dfc9ea1e30ff to PENDING with url http://10.15.247.135:8081/job/ghpqb-sql-test/44/ and message: "Build started sha1 is merged."
Setting status of 716511a912e57e383299fb1777d9dfc9ea1e30ff to PENDING with url http://10.15.247.135:8081/job/ghpqb-sql-test/44/ and message: "Build started sha1 is merged."
ビルドします。 ワークスペース: /usr/local/opt/jenkins/jobs/ghpqb-sql-test/workspace
 > git rev-parse --is-inside-work-tree # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url https://github.com/boyd-mask/git-jenkins-test/ # timeout=10
Fetching upstream changes from https://github.com/boyd-mask/git-jenkins-test/
 > git --version # timeout=10
 > git fetch --tags --progress https://github.com/boyd-mask/git-jenkins-test/ +refs/heads/*:refs/remotes/origin/* +refs/pull/*:refs/remotes/origin/pr/*
 > git rev-parse refs/remotes/origin/pr/4/merge^{commit} # timeout=10
 > git rev-parse refs/remotes/origin/origin/pr/4/merge^{commit} # timeout=10
Checking out Revision a903aebd223aa19489f620a8440306087a8015f5 (refs/remotes/origin/pr/4/merge)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f a903aebd223aa19489f620a8440306087a8015f5
First time build. Skipping changelog.
[workspace] $ /bin/sh -xe /tmp/hudson5820544450912319342.sh
+ /usr/local/bin/fab -f /usr/local/jenkins/fabfile.py RunYaml:oauth_token=********,github_api_url=https://api.github.com,owner=boyd-mask,repo=git-jenkins-test,git_current_branch=origin/pr/4/merge,git_current_commit_id=a903aebd223aa19489f620a8440306087a8015f5
Python-2.7.9
bin
etc
games
include
jenkins
lib
lib64
libexec
nz
opt
ruby-2.2.0
sbin
share
src
[localhost] local: curl -H "Authorization: token ********" https://api.github.com/repos/boyd-mask/git-jenkins-test/commits/a903aebd223aa19489f620a8440306087a8015f5
git_head_commit_id: 716511a912e57e383299fb1777d9dfc9ea1e30ff
[localhost] local: curl -H "Authorization: token ********" https://api.github.com/repos/boyd-mask/git-jenkins-test/commits/716511a912e57e383299fb1777d9dfc9ea1e30ff
TEST COMMIT
/usr/local
[localhost] local: ls /usr/local
/usr/local


Done.
Adding one-line test results to commit status...
Setting status of 716511a912e57e383299fb1777d9dfc9ea1e30ff to SUCCESS with url http://10.15.247.135:8081/job/ghpqb-sql-test/44/ and message: "YEEEEEEEEESSSSSSSS!
 No test results found."
Setting status of 716511a912e57e383299fb1777d9dfc9ea1e30ff to SUCCESS with url http://10.15.247.135:8081/job/ghpqb-sql-test/44/ and message: " "
Warning: you have no plugins providing access control for builds, so falling back to legacy behavior of permitting any downstream builds to be triggered
[8mha:AAAApx+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAzREgZp/az8JP3k/Nzc1LwS3aLU4tKcEt30zJKM0iR9APyWVhPQAAAA[0mcomment-result-github の新規ビルドを実行します
Finished: SUCCESS
```

byam committed:
TEST TEST TEST
/usr/local/opt/jenkins/jobs

byam commented:
bot test this

bot-mask commented:
Hello Guys. I am Mr.Robot. 

```
GitHub pull request #4 of commit c3bacdb545c0e7f2f16a474383fc23cdfa1ba093, no merge conflicts.
Setting status of c3bacdb545c0e7f2f16a474383fc23cdfa1ba093 to PENDING with url http://10.15.247.135:8081/job/ghpqb-sql-test/45/ and message: "Build started sha1 is merged."
Setting status of c3bacdb545c0e7f2f16a474383fc23cdfa1ba093 to PENDING with url http://10.15.247.135:8081/job/ghpqb-sql-test/45/ and message: "Build started sha1 is merged."
ビルドします。 ワークスペース: /usr/local/opt/jenkins/jobs/ghpqb-sql-test/workspace
 > git rev-parse --is-inside-work-tree # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url https://github.com/boyd-mask/git-jenkins-test/ # timeout=10
Fetching upstream changes from https://github.com/boyd-mask/git-jenkins-test/
 > git --version # timeout=10
 > git fetch --tags --progress https://github.com/boyd-mask/git-jenkins-test/ +refs/heads/*:refs/remotes/origin/* +refs/pull/*:refs/remotes/origin/pr/*
 > git rev-parse refs/remotes/origin/pr/4/merge^{commit} # timeout=10
 > git rev-parse refs/remotes/origin/origin/pr/4/merge^{commit} # timeout=10
Checking out Revision 4e7f79d02b40c4d9c7f7cb606fdfeeffbe48101a (refs/remotes/origin/pr/4/merge)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f 4e7f79d02b40c4d9c7f7cb606fdfeeffbe48101a
First time build. Skipping changelog.
[workspace] $ /bin/sh -xe /tmp/hudson7689403674206875180.sh
+ /usr/local/bin/fab -f /usr/local/jenkins/fabfile.py RunYaml:oauth_token=********,github_api_url=https://api.github.com,owner=boyd-mask,repo=git-jenkins-test,git_current_branch=origin/pr/4/merge,git_current_commit_id=4e7f79d02b40c4d9c7f7cb606fdfeeffbe48101a
comment-result-github
ghpqb-sql-test
[localhost] local: curl -H "Authorization: token ********" https://api.github.com/repos/boyd-mask/git-jenkins-test/commits/4e7f79d02b40c4d9c7f7cb606fdfeeffbe48101a
git_head_commit_id: c3bacdb545c0e7f2f16a474383fc23cdfa1ba093
[localhost] local: curl -H "Authorization: token ********" https://api.github.com/repos/boyd-mask/git-jenkins-test/commits/c3bacdb545c0e7f2f16a474383fc23cdfa1ba093
TEST TEST TEST
/usr/local/opt/jenkins/jobs
[localhost] local: ls /usr/local/opt/jenkins/jobs
/usr/local/opt/jenkins/jobs


Done.
Adding one-line test results to commit status...
Setting status of c3bacdb545c0e7f2f16a474383fc23cdfa1ba093 to SUCCESS with url http://10.15.247.135:8081/job/ghpqb-sql-test/45/ and message: "YEEEEEEEEESSSSSSSS!
 No test results found."
Setting status of c3bacdb545c0e7f2f16a474383fc23cdfa1ba093 to SUCCESS with url http://10.15.247.135:8081/job/ghpqb-sql-test/45/ and message: " "
Warning: you have no plugins providing access control for builds, so falling back to legacy behavior of permitting any downstream builds to be triggered
[8mha:AAAApx+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAzREgZp/az8JP3k/Nzc1LwS3aLU4tKcEt30zJKM0iR9APyWVhPQAAAA[0mcomment-result-github の新規ビルドを実行します
Finished: SUCCESS
```

byam committed:
CONFIGS TEST COMMIT
/usr/local/opt/jenkins/jobs
/usr/local/opt/jenkins/jobs/bya

byam commented:
test this please

byam commented:
test this please.

bot-mask commented:
Hello Guys. I am Mr.Robot. 

```
GitHub pull request #4 of commit 6b7a33d3b34b7913e0dc7b103c95cfdec5189fde, no merge conflicts.
Setting status of 6b7a33d3b34b7913e0dc7b103c95cfdec5189fde to PENDING with url http://10.15.247.135:8081/job/ghpqb-sql-test/34/ and message: "Build started sha1 is merged."
Setting status of 6b7a33d3b34b7913e0dc7b103c95cfdec5189fde to PENDING with url http://10.15.247.135:8081/job/ghpqb-sql-test/34/ and message: "Build started sha1 is merged."
ビルドします。 ワークスペース: /usr/local/opt/jenkins/jobs/ghpqb-sql-test/workspace
 > git rev-parse --is-inside-work-tree # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url https://github.com/boyd-mask/git-jenkins-test/ # timeout=10
Fetching upstream changes from https://github.com/boyd-mask/git-jenkins-test/
 > git --version # timeout=10
 > git fetch --tags --progress https://github.com/boyd-mask/git-jenkins-test/ +refs/heads/*:refs/remotes/origin/* +refs/pull/*:refs/remotes/origin/pr/*
 > git rev-parse refs/remotes/origin/pr/4/merge^{commit} # timeout=10
 > git rev-parse refs/remotes/origin/origin/pr/4/merge^{commit} # timeout=10
Checking out Revision 15e9285549a339514d28b5188304e76d63fc8d4d (refs/remotes/origin/pr/4/merge)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f 15e9285549a339514d28b5188304e76d63fc8d4d
First time build. Skipping changelog.
[workspace] $ /bin/sh -xe /tmp/hudson5093126375816775146.sh
+ /usr/local/bin/fab -f /usr/local/jenkins/fabfile.py CurrentCommitMessage:oauth_token=6d85708aa08c6d12c6fedab3b600424ada445df3,github_api_url=https://api.github.com,owner=boyd-mask,repo=git-jenkins-test,git_current_branch=origin/pr/4/merge,git_current_commit_id=15e9285549a339514d28b5188304e76d63fc8d4d
[localhost] local: curl -H "Authorization: token 6d85708aa08c6d12c6fedab3b600424ada445df3" https://api.github.com/repos/boyd-mask/git-jenkins-test/commits/15e9285549a339514d28b5188304e76d63fc8d4d
Traceback (most recent call last):
  File "/usr/local/Python-2.7.9/lib/python2.7/site-packages/fabric/main.py", line 743, in main
    *args, **kwargs
  File "/usr/local/Python-2.7.9/lib/python2.7/site-packages/fabric/tasks.py", line 405, in execute
    results["<local-only>"] = task.run(*args, **new_kwargs)
  File "/usr/local/Python-2.7.9/lib/python2.7/site-packages/fabric/tasks.py", line 171, in run
    return self.wrapped(*args, **kwargs)
  File "/usr/local/jenkins/fabfile.py", line 19, in CurrentCommitMessage
    git_head_commit_id = json.loads(str(commit_raw))["parents"][-1]["sha"]
KeyError: "parents"
Build step "シェルの実行" marked build as failure
Adding one-line test results to commit status...
Setting status of 6b7a33d3b34b7913e0dc7b103c95cfdec5189fde to FAILURE with url http://10.15.247.135:8081/job/ghpqb-sql-test/34/ and message: "FAILED!
 No test results found."
Setting status of 6b7a33d3b34b7913e0dc7b103c95cfdec5189fde to FAILURE with url http://10.15.247.135:8081/job/ghpqb-sql-test/34/ and message: "failure
 "
Warning: you have no plugins providing access control for builds, so falling back to legacy behavior of permitting any downstream builds to be triggered
ha:AAAApx+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAzREgZp/az8JP3k/Nzc1LwS3aLU4tKcEt30zJKM0iR9APyWVhPQAAAAcomment-result-github の新規ビルドを実行します
Finished: FAILURE

```

byam committed:
WHAT
/usr/local/jenkins

byam commented:
test this please.

boyd-mask commented:
add to whitelist

boyd-mask commented:
ok to test

boyd-mask commented:
test this please.

byam commented:
test this please.

byam commented:
ok to test

boyd-mask commented:
ok to test

byam committed:
WHYYYYYYYYYYYYYYYYYYYYYYYYYYY
/usr/local/opt/jenkins

byam commented:
test this please.

bot-mask commented:
Can one of the admins verify this patch?

bot-mask commented:
Can one of the admins verify this patch?
